### PR TITLE
John Morrow

### PR DESCRIPTION
**Pull request by CodeMorrow**

### CODE REVIEW

File: lib/hashes.md [hunk 1]
```diff
@@ -5,40 +5,43 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 # How do you add information to a Hash?
 
-Replace me with your response...
+my_hash_name[new_key_name]= new_value
 
 # How would you perform an operation on every element inside a Hash?
 
-Replace me with your response...
+my_hash.each do |key, value|
+    my_hash[key] = new_value (whatever edit I want to make for every value)
```
Comment (halfghaninne) [inline lib/hashes.md:31]: Don't forget to close out the `each` block with an `end`!

File: lib/hashes.md [hunk 2]
```diff
@@ -5,40 +5,43 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 
 # How would you change the value of a particular element in a Hash?
 
-Replace me with your response...
+my_hash_name[particular_key]= new_value
 
 # How do you delete an element from a Hash?
 
-Replace me with your response...
+my_hash_name.delete(element)
```
Comment (halfghaninne) [inline lib/hashes.md:39]: How do you reference the element?

**Comment by halfghaninne** [comment]
Otherwise, 👍

File: lib/hashes.md (hunk 2 above)
Comment (CodeMorrow) [inline lib/hashes.md:39]: by calling the name of the key. I should have replaced the word element with key.

**Pull request closed without merging**